packages/sass-variables/_typography.scss

```diff
@@ -217,21 +275,41 @@ $puppet-type-code-small-font-weight: $puppet-type-weight-normal !default;
   font-size: $puppet-type-code-font-size;
   font-weight: $puppet-type-code-font-weight;
   line-height: $puppet-type-code-line-height;
+  background-color: $puppet-n100;
+  padding: 0 $puppet-common-spacing-base;
+  border-radius: 2px;
 }
 
 @mixin puppet-type-code-small() {
+  @include puppet-type-code();
   font-family: $puppet-type-code-small-font-family;
   font-size: $puppet-type-code-small-font-size;
   font-weight: $puppet-type-code-small-font-weight;
   line-height: $puppet-type-code-small-line-height;
 }
 
+// Same as puppet-type-code, but scale the font size to that of the surrounding
+// text (based on the ratio of default body & code font sizes)
+@mixin puppet-type-code-scaled() {
+  @include puppet-type-code();
+  font-size: #{$puppet-type-code-font-size / $puppet-type-body-font-size}em;
+  line-height: inherit;
+}
+
 @mixin puppet-type-code-block() {
-  background-color: $puppet-n100;
+  background-color: $puppet-n50;
   box-shadow: inset 8px 0 0 #e4e9ed;
+  box-sizing: border-box;
   display: block;
+  @include puppet-type-code();
+  line-height: $puppet-type-code-block-line-height;
+  overflow-x: auto;
   padding: $puppet-common-spacing-base * 3 $puppet-common-spacing-base * 6;
-  white-space: pre;
+  white-space: pre-wrap;
```

Comment:
Code blocks seemed to be wrapping in some contexts and not in others - I chose wrap here, but open to changing it of course